src/modules/module_26700.c

```diff
@@ -0,0 +1,270 @@
+/**
+ * Author......: See docs/credits.txt
+ * License.....: MIT
+ */
+
+#include "common.h"
+#include "types.h"
+#include "modules.h"
+#include "bitops.h"
+#include "convert.h"
+#include "shared.h"
+
+static const u32   ATTACK_EXEC    = ATTACK_EXEC_OUTSIDE_KERNEL;
+static const u32   DGST_POS0      = 0;
+static const u32   DGST_POS1      = 1;
+static const u32   DGST_POS2      = 2;
+static const u32   DGST_POS3      = 3;
+static const u32   DGST_SIZE      = DGST_SIZE_4_4;
+static const u32   HASH_CATEGORY  = HASH_CATEGORY_RAW_CIPHER_KPA;
```

Comment:
Is not a RAW Cipher, I think FDE fits best

Reply:
or EAS ?

Reply:
We can also make a new category. I just find it a bit lame when there's only one entry per category. But is for sure not EAS. As far as I understand this is a block device which is getting encrypted. And that's what we call a full disc encryption.

Reply:
yes, vbox is coming bro ;) I will add a new category later

Reply:
OK, lets not forget to add it at a later point in time when we have more plugins for that new category.